core/test/CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ execute_process(
     COMMAND "${Python_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/old_names.py"
     WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
 )
+execute_process(
+    COMMAND "${Python_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/ParaGridIO_input_test.py"
+    WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+)
```

Comment:
I noticed that `ParaGridIO_input_test.nc` was included as a NetCDF file so dropped it and set things up so that it gets generated during the build.